manager/README.md

```diff
@@ -55,6 +55,14 @@ VAULT_MODULES_ROLE="module"
 ENABLE_WEBHOOKS="false"
 ```
 
+If the manager runs in the Razee environment, you also need to add the following environment variables:
+
+```bash
+RAZEE_USER="razee-dev@example.com"
+RAZEE_PASSWORD="password123"
+RAZEE_URL="http://localhost:3333/graphql"
```

Comment:
Where is the port-forward from razee pod to localhost:3333?

Reply:
Don't need a "port-forward".
There is a NodePort service "razeedash-api-np" created by https://github.com/fybrik/fybrik/blob/ee85999dd766b97feb0ab2f7e095247deccb580f/third_party/razee/razee-nodeports.yaml#L5

Reply:
OK, thanks!